Added command for definitions and synomyms of words

Crippius opened the pull request:
I added two programs that make it so that if the user wants to, he can search for the definition of words or its synonyms

virejdasani commented:
Hi @Crippius, thanks for contributing! Before I can merge, can you add the 2 commands you added, in the `res/AllCommands.txt` file

virejdasani commented:
Also, a ⭐  to the repo is much appreciated :)

Crippius commented:
Hey there, I just updated the res/AllCommands.txt file to include the commands I added and starred the repo, hopefully know it's all good for merging :)
